### ISSUE TITLE
my experience

### ISSUE THREAD

**Issue by Bia10** [libgenapps/LibgenDesktop#15]
hello, 

Regarding the app.

Scan log - missing formatting , it would be great to know what error(path too long? access rights? being used?) actually happen
Scan algorithm - very slow, constantly keeps triggering false positives

the false positives have following in common:

 libgenID: 83308
 filesize: 0.00 bytes
 comments: md5 файла нулевого размера 

this app seems to have a long way to go thanks for yours work

**Comment by libgenapps** [comment]
That's right, the scanning functionality is implemented on a proof-of-concept level only. I am planning to complete it in the upcoming 1.4 release.

**Comment by libgenapps** [comment]
This feature is postponed until 1.5.
